### DISCUSSION TITLE
Is there a way to start the hotspot without the pi being connected to internet?

### DISCUSSION THREAD

**Discussion by Soulsender** [RaspAP/raspap-webgui#1296] [Q&A]
I want to have my pi working as an access point that my phone can connect to so I can remotely ssh into it on-the-go. The problem is that hostapd will only start if it is connected to internet via ethernet cable. Is there a way I can turn this off so the hotspot will start, but will serve me no internet?

**Comment by billz** [comment]
hostapd is capable of starting in an "offline" mode (ie., without ethernet or any internet connectivity). RaspAP has been used in many instances to provide communication between hosts on a purely local wireless network (here's [one example](https://discuss.bluerobotics.com/t/subsea-resident-auv/9771)). If your service is failing to start, search this project's resources on debugging hostapd.
